packages/console/package.json

```diff
@@ -7,43 +7,43 @@
   "scripts": {
     "dev": "vite",
     "watch": "vite",
-    "build": "vite build"
+    "build": "vite build",
+    "test": "vitest"
   },
   "dependencies": {
     "@antv/g2": "4.1.23",
     "cronstrue": "2.5.0",
     "lodash.clonedeep": "4.5.0",
-    "mobx": "6.4.2",
-    "mobx-react": "7.3.0",
+    "mobx": "6.13.5",
+    "mobx-react": "9.1.1",
     "multer": "1.4.5-lts.1",
-    "polished": "4.1.3",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-helmet": "6.1.0",
-    "react-markdown": "7.0.0",
-    "react-router": "5.2.0",
-    "react-router-dom": "5.2.0",
+    "polished": "4.3.1",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
+    "react-helmet-async": "2.0.5",
```

Comment:
Replaced `react-helmet` (kind of abandoned) with this fork since it breaks stuff with peer dependency on older React